go.mod

```diff
@@ -18,13 +18,13 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.12.1
 	github.com/rancher/apiserver v0.0.0-20210922180056-297b6df8d714
-	github.com/rancher/dynamiclistener v0.2.1-0.20200714201033-9c1939da3af9
+	github.com/rancher/dynamiclistener v0.3.5
```

Comment:
This brings in 40 new commits

```
$ git log --no-merges --oneline 9c1939da3af9..v0.3.5 | wc -l
40
```

I don't want to create backport issues for all those changes but we might want to somehow give QA a headsup around this